Modix.Bot/Modules/MarkerRoleModule.cs

```diff
@@ -5,21 +5,53 @@
 using Discord.Commands;
 using Modix.Services.Core;
 using Modix.Data.Models.Core;
+using Humanizer;
+using Discord.WebSocket;
 
 namespace Modix.Modules
 {
-    [Group("pingrole"), Name("Marker Role Manager"), Summary("Allows you to add and remove specific marker roles.")]
+    [Group("pingrole")]
+    [Alias("pingroles")]
+    [Name("Marker Role Manager")]
+    [Summary("Allows you to add and remove specific marker roles.")]
     public class MarkerRoleModule : ModuleBase
     {
-        private readonly IAuthorizationService _authorizationService;
         private readonly IDesignatedRoleService _designatedRoleService;
 
-        public MarkerRoleModule(IAuthorizationService authorizationService, IDesignatedRoleService designatedRoleService)
+        public MarkerRoleModule(IDesignatedRoleService designatedRoleService)
         {
-            _authorizationService = authorizationService;
             _designatedRoleService = designatedRoleService;
         }
 
+        [Command("list")]
+        [Alias("")]
+        public async Task List()
+        {
+            var pingRoles = await _designatedRoleService
+                .SearchDesignatedRolesAsync(new DesignatedRoleMappingSearchCriteria()
+                {
+                    Type = DesignatedRoleType.Pingable
+                });
+
+            var pingRolesInformation = pingRoles.Select(x =>
```

Comment:
We should probably do an orderby on the role name so that entries show up in a predictable order

Reply:
Very true, fixed.